src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -116,6 +120,41 @@ def start_monitoring(self):
                                   str(exception))
                 self.logger.error(traceback.format_exc())
 
+    def _read_and_send_message(self, file_name, loop_start_time):
+        """Attempt to build a message and send it, re-raising the SenderKeyExpired and
+        NoUsableKeysException exceptions.
```

Comment:
Oh, I see, you commented about it up here. Hmm... The main point of this method isn't to reraise the exceptions but to have its own exception handler. I would mention that first.